docs/src/lib/sets/Zonotope.md

```diff
@@ -15,6 +15,8 @@ ngens(::Zonotope)
 togrep(::Zonotope)
 reduce_order(::Zonotope, ::Union{Integer, Rational})
 split(::Zonotope, ::Int)
+split(Z::Zonotope, gens::AbstractVector, n::AbstractVector)
+split(Z::Zonotope{N, SVector{n, N}, <:SMatrix{n, p, N}}, j::Int) where {N, n, p}
```

Comment:
there is a docs build error, https://travis-ci.org/github/JuliaReach/LazySets.jl/jobs/706883297#L557

doing `using StaticArrays` in the docs setup may help. otherwise, just remove line 19..

Reply:
That would be here I guess:
https://github.com/JuliaReach/LazySets.jl/blob/a581de8c90d8e3c6d9689a7d724f9c77d7347225/docs/make.jl#L3

Reply:
Hm, I just tried that and it still gives the same error.

Reply:
> otherwise, just remove line 19

I agree, remove that line and the documentation for that method because it is actually identical to the old one I think.

Reply:
yes, indeed

Reply:
i've removed the docs for sarrays since its a dup of the existing one.